knowledge_graph/routes.py

```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
 
     response = Response(dumps(value_scores))
     return response, 200
+    
+
+@app.route('/personal_values', methods=['GET'])
+def get_personal_values():
+    """ Given a session-id, this returns the top three personal values for a user
+    
+    """
+    session_id = int(request.args.get('session-id'))
+    
+    try:
+        scores = db.session.query(Scores).filter_by(session_id=1).first()
```

Comment:
Why are we filtering by session_id = 1 here? shouldn't it be by the session id provided?

Reply:
Whoops, I forgot to change that back. I'll fix.